testonly/hammer/hammer.go

```diff
@@ -284,6 +285,75 @@ func newWorker(cfg *MapConfig, prng *rand.Rand) *mapWorker {
 	}
 }
 
+// readWorker performs read-only operations on a fixed map.
+type readWorker struct {
+	*mapWorker
+
+	validReadOps   *validReadOps
+	invalidReadOps *invalidReadOps
+}
+
+func newReadWorker(s *hammerState, idx int) *readWorker {
+	return &readWorker{
+		mapWorker: newWorker(s.cfg, rand.New(rand.NewSource(int64(idx)))),
+
+		validReadOps:   s.validReadOps,
+		invalidReadOps: s.invalidReadOps,
+	}
+}
+
+func (w *readWorker) loopUntilDone(ctx context.Context, done <-chan struct{}) error {
```

Comment:
How about just `run`, same as in `writeWorker`?

Reply:
Resolved.